cli/ballerina-cli/src/main/java/io/ballerina/cli/utils/NativeUtils.java

```diff
@@ -169,9 +169,7 @@ public static void createReflectConfig(Path nativeConfigPath, Package currentPac
             if (mockedFunctionClassFile.isFile()) {
                 BufferedReader br = Files.newBufferedReader(mockedFunctionClassPath, StandardCharsets.UTF_8);
                 Gson gsonRead = new Gson();
-                Map<String, String[]> testFileMockedFunctionMapping = gsonRead.fromJson(br,
-                        new TypeToken<Map<String, String[]>>() {
-                        }.getType());
+                Map<String, String[]> testFileMockedFunctionMapping = gsonRead.fromJson(br, new TypeToken<>() { });
```

Comment:
```suggestion
                Map<String, String[]> testFileMockedFunctionMapping = gsonRead.fromJson(br, new TypeToken<>() {});
```
The white space is not needed (to make the code more compact). Shall we remove in other places as well?

Reply:
@HindujaB This leads to a checkstyle error: `'{' is not followed by whitespace.`